### PR TITLE
Feature/tensorboards

### PR DESCRIPTION
**Pull request by timdesmetML6**
This PR does two things: 
- it renames the folder for the engineering chapter (previously the MLIP and SA chapters)
- it adds a quick tip about [this blog post](https://blog.ml6.eu/a-vertex-ai-tensorboard-alternative-for-smaller-budgets-part-2-923953c1e422)

### CODE REVIEW

File: engineering/2022_02_24_sharable_tensorboards/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Any other alternatives? 🤷‍♂️
+
+Some other options (like TensorBoard.dev)  make your data publicly available (it goes without saying but this is a big nono when working with customer data 🙅‍♂️)
+Other solutions require other people to set up a local environment to run TensorBoard and thus require certain technical skills from those people.
+
+## Our solution 🦾
+
+You can use the Terraform code in the `tensorboard.tf` file to generate your own securely sharable TensorBoard for a fraction of the price. 
+
+Some minor changes have to be made to the file:
```
Comment (Ruwann) [inline engineering/2022_02_24_sharable_tensorboards/README.md:17]: These are now stored in variables, so the description here isn't entirely accurate. I think we can just reword to something like "There are some variables you have to supply:" - WDYT?

**Review by Ruwann — APPROVED** [review]
LGTM! Left one comment that would be nice to change as well, if you agree :)

File: engineering/2022_02_24_sharable_tensorboards/README.md (hunk 1 above)
Comment (timdesmetML6) [reply engineering/2022_02_24_sharable_tensorboards/README.md]: Nice catch! Added the change just now!

**Review by Ruwann — APPROVED** [review]
Awesome, thanks!

**Pull request merged**